cli/pcluster/config/pcluster_config.py

```diff
@@ -456,10 +464,11 @@ def __ec2_run_instance(self, max_size, **kwargs):
                     "available in the Compute subnet.\n{1}".format(max_size, message)
                 )
             elif code == "InvalidParameterCombination":
-                self.error(message)
+                self.error("Unable to check AWS Account capacity. {0}".format(message))
```

Comment:
Minor - Is there a reason this error message is included on the same line whereas in the other cases (except for the UnsupportedOperation one) the error message is printed on the next line?

Reply:
fixed